lab1.gs

```diff
@@ -0,0 +1,29 @@
+function myFunction() {
+  let excel = SpreadsheetApp.getActiveSpreadsheet();
+  let sheet = excel.getSheetByName("Sheet1");
+  let range = sheet.getRange("B2:E5");
```

Comment:
What if the sheet has entries beyond the 5th row? Please handle this case as well.

Reply:
Sure.